TurkishFilter/sections/specific.txt

```diff
@@ -41,6 +41,7 @@ yemektarifleri.org,lifebursa.com,mshowto.org,macizlee1.com,macizlee.ne,dunyaatla
 ! ###adStarter
 fullhdfilmcibaba.com,720pizlemek.com,1080pfilmizle.net,fullhdfilmcidayi.com,hdfilmizledayi.com,720-pizleme.com,fullhdfilmizlebaba.com,evrenselfilmlerim.net###adStarter
 !
+pazarlamasyon.com##img[src="https://pazarlamasyon.com/wp-content/uploads/2019/01/3000x500-Duyuru-1.jpg"]
```

Comment:
This rule looks better:
`pazarlamasyon.com##a[href="https://www.kobieticaretzirvesi.com/"] > img`

Reply:
Single ad, so just `||pazarlamasyon.com/wp-content/uploads/*/3000x500-Duyuru-*.jpg`.

Reply:
Or `||pazarlamasyon.com/wp-content/uploads/*/3000x500-`